common/src/java/org/apache/hadoop/hive/conf/HiveConf.java

```diff
@@ -3060,8 +3060,15 @@ private static void populateLlapDaemonVarsSet(Set<String> llapDaemonVarsSetLocal
         "Creates an _orc_acid_version file along with acid files, to store the version data"),
 
     HIVE_TXN_READONLY_ENABLED("hive.txn.readonly.enabled", false,
-      "Enables read-only transaction classification and related optimizations"),
-
+        "Enables read-only transaction classification and related optimizations"),
+
+    HIVE_ACID_LOCKLESS_READS_ENABLED("hive.acid.lockless.reads.enabled", false,
```

Comment:
Where will we document the requirements when this lockless reads can work?
Like `HIVE_ACID_TRUNCATE_USE_BASE` should be true

Reply:
HIVE_ACID_LOCKLESS_READS_ENABLED is independent from HIVE_ACID_TRUNCATE_USE_BASE. HIVE_ACID_LOCKLESS_READS_ENABLED enables group of non-blocking operations at once like drop/truncate table/partition.